### DISCUSSION TITLE
Custom `--emails-dir` config

### DISCUSSION THREAD

**Discussion by edgarlr** [sofn-xyz/mailing#113] [Q&A] [answered]
Are there any specific reasons to don't allow custom emails dir config for both the generator and the preview server? 

I believe it currently only supports `./email` and `./src/emails` but I'm using it as a package inside a monorepo and looks a bit redundent to have the file scrutrue with nested emails folders

```
packages
├── emails
│   ├── emails // or (src/emails)
│   └── // All mailing generated files

```

**Comment by psugihara** [comment]
Hey @edgarlr. Only reason currently is that we don't have a way to store the path. After we add the config file, it should be trivial. Sorry about the inconvenience until then though. 

Here's the config file issue (will be worked on soon) https://github.com/sofn-xyz/mailing/issues/74

cc @alexfarrill

**Reply by psugihara** [reply]
Actually I guess we could add before config is available. It would just have a sharp edge:

If you forget to pass the arg, you might re-init and creat an additional emails directory. 

That seems like no big deal as passing the arg will be kind of a pro-level feature anyway. Either way, both will be added soon. Appreciate the discussion @edgarlr!

**Reply by psugihara** [reply]
https://github.com/sofn-xyz/mailing/issues/115

**Reply by edgarlr** [reply]
Hey @psugihara ! This sounds great! Thanks for your comments. the general approach sounds good! Thanks again!

**Comment by psugihara** [comment, marked as answer]
Quick followup... @alexfarrill shipped the solution in #116 !

The emails directory can now be specified in mailing.config.json or with the `--emails-dir` CLI option.

The change has been release in the latest version of `mailing` and `mailing-core`.

**Reply by edgarlr** [reply]
@psugihara @alexfarrill Awesome! this is great to hear! thanks for jumping on it so quickly!

**Reply by psugihara** [reply]
no problem, thanks for surfacing the bug!

if you have time, i'd love to hear a bit more about how you're using or thinking about using mailing (https://github.com/sofn-xyz/mailing/discussions/121)

**Reply by edgarlr** [reply]
For sure! I'll make sure to add my comment there!

**Reply by psugihara** [reply]
awesome, thx so much
